<?xml version="1.0" encoding="UTF-8"?>
<launch>
   <include file="$(find uv_worlds)/launch/lake.launch"/>
   
   <arg name="node_start_delay" value="5.0" />

   <include file="$(find vehicle_descriptions)/launch/vtec_u3.launch"/>
   
   <arg name="node_start_delay_2" default="1.0" /> 

   <include file="$(find uv_worlds)/launch/task_obstacles.launch"/>
   
   <arg name="node_start_delay_3" default="1.0" /> 

   <include file="$(find vanttec_uuv)/launch/uuv_simulation.launch"/>

   <arg name="node_start_delay_4" default="10.0" /> 
   
   <node pkg="vehicle_descriptions" type="gazebo_interface" name="gazebo_interface"/>   

   <node pkg="vanttec_uuv" type="object_detector_node" name="object_detector_node"/>   

   </launch>